src/librbd/cache/ParentCacheObjectDispatch.cc

```diff
@@ -115,7 +115,8 @@ bool ParentCacheObjectDispatch<I>::read(
 
   m_cache_client->lookup_object(m_image_ctx->data_ctx.get_namespace(),
                                 m_image_ctx->data_ctx.get_id(),
-                                (uint64_t)snap_id, oid, std::move(ctx));
+                                (uint64_t)snap_id, object_len, oid,
```

Comment:
This should be the fixed object size -- not the current IO length.

Reply:
so sorry. i'm still not clear what's means for current IO length. could you please give some detail explain. many thanks.

Reply:
RBD had 4MiB default objects sizes. If I request a 1 byte IO, we still might need to promote a 4MiB object -- so the throttle should be based on the object size, not the IO request size.

Reply:
thanks jason, but how do the throttle know object size ?

Reply:
`m_image_ctx.get_object_size()`